src/app/pages/ArticlePage/ReadMoreButton/index.styles.ts

```diff
@@ -11,7 +11,7 @@ export default {
       display: 'block',
       width: 'calc(100% - 2rem)',
       padding: `${spacings.DOUBLE}rem 0`,
-      margin: `${spacings.QUADRUPLE}rem ${spacings.DOUBLE}rem -0.5rem  ${spacings.DOUBLE}rem`,
+      margin: `${spacings.QUADRUPLE}rem ${spacings.DOUBLE}rem -1.5rem  ${spacings.DOUBLE}rem`,
```

Comment:
The bottom of the `<main>` container, which contains the button, has a padding of 24px. The `<aside>` underneath which contains the related content component has a top margin of 16px. The Related Content heading has a margin top of 16px. To achieve the gap of 32px between the button and the related content component I have added a minus margin of 1.5rem. 

Is it always the related content heading first? Do all headings have this margin top?